NEXT_RELEASE.md

```diff
@@ -0,0 +1,370 @@
+# Next Major Release - Developer Guide
+
+This guide explains how to contribute breaking changes to the next major version of the Polymesh SDK using the ephemeral integration system.
+
+## 📋 Overview
+
+Breaking changes for the next major release are managed through special branches that follow a specific naming convention. These branches are automatically discovered, validated, and integrated to create preview releases.
+
+## 🌿 Branch Naming Convention
+
+### Format
+```
+bc-<number>-<description>
+```
+
+### Rules
+- **Must start with `bc-`**
+- **Include a number** (determines integration order)
```

Comment:
I am not a fan of numbers being chosen by developers like this. Commits should be apliable in arbitrary order for the most part. 

Pragmatically I don't think this will be an issue, but it feels like its asking for collision or coordination. It feels like its adding in a distributed computing problem of sorts.